reduction/tests/test_reduction.py

```diff
@@ -56,17 +56,29 @@ def test_q_summing(nexus_dir):
         ws_sc = mtd_api.Load("REF_L_%s" % 198415)
     qz_mid0, refl0, _, meta_data = template.process_from_template_ws(ws_sc, template_path, info=True)
 
-    assert np.fabs(meta_data["dq_over_q"] - 0.02759) < 1e-3
+    assert np.fabs(meta_data["dq_over_q"] - 0.02261) < 1e-3
```

Comment:
This would suggest `dq` has decreased significantly (assuming `q` is the same). Is this reasonable?